src/agent/onefuzz-agent/src/local/tui.rs

```diff
@@ -229,6 +317,49 @@ impl TerminalUi {
                 .constraints([Constraint::Percentage(25), Constraint::Percentage(75)].as_ref())
                 .split(f.size());
 
+            let log_area = chunks[1];
+            let top_area = Layout::default()
+                .direction(Direction::Horizontal)
+                .constraints([Constraint::Percentage(50), Constraint::Percentage(50)].as_ref())
+                .split(chunks[0]);
+
+            let file_count_area = top_area[0];
+            let coverage_area = Layout::default()
+                .direction(Direction::Vertical)
+                .constraints([Constraint::Percentage(25), Constraint::Percentage(75)].as_ref())
+                .margin(1)
+                .split(top_area[1]);
+
+            let label = format!("{:.2}%", rate * 100.0);
+
+            let coverage_block = Block::default().borders(Borders::ALL).title("Coverage:");
+
+            f.render_widget(coverage_block, top_area[1]);
+
+            let gauge = Gauge::default()
+                .gauge_style(
+                    Style::default()
+                        .fg(Color::Magenta)
```

Comment:
Can we pick a different color scheme?  Perhaps no color scheme at all, but just the bar graph using inverted terminal defaults?  

This has accessibility issues for people who are low vision.